src/Features/Core/Portable/InitializeParameter/AbstractAddParameterCheckCodeRefactoringProvider.cs

```diff
@@ -29,66 +30,116 @@ internal abstract partial class AbstractAddParameterCheckCodeRefactoringProvider
         where TExpressionSyntax : SyntaxNode
         where TBinaryExpressionSyntax : TExpressionSyntax
     {
+
         protected override async Task<ImmutableArray<CodeAction>> GetRefactoringsAsync(
```

Comment:
Let's try to combine the logic of two `GetRefactoringsAsync` into one, so we can avoid making any change in the base type `AbstractInitializeParameterCodeRefactoringProvider` (so it won't affect other related features)